blog/smart-testing-0.0.1.textile

```diff
@@ -0,0 +1,70 @@
+
+p. This time it's not a new extension to Arquillian Testing Platform, but a brand new tool which we hope will bring a breath of fresh air into your builds, let it be on your local machine or the CI server.
+
+p. You probably know it best - you write some code and tests, run the build, wait few minutes to only see at the later stage that your changes are breaking it. Of course you can go and grab a coffee, but if you commit early and often that might mean caffeine overdose in a very short time. Running tests before pushing is a good practice, but if it takes too long it simply slows you down. 
+
+Moreover having long lasting builds on your CI server makes your CI/CD process not really that continuous and leads to Pull Requests piling up. Bringing new features to production with confidence should be seamless. And for this we need to have faster feedback loops. Smart Testing is a tool that speeds up the test running phase by reordering test execution plan to increase a probability of fail-fast execution and thus give you faster feedback about your project’s health.
+
+That's why we created a cure for you!
+
+h3. Higlights of this release
+
+p. With this release we provide following strategies which you can use to optimize your build execution:
+
+  * @new@ - gives higher priority to newly added tests
+  * @changed@ - gives higher priotity to changed tests (e.g. new test methods or refactorings of components under tests)
+  * @affected@ - based on changes in your business logic gives higher priority to tests which are excercising them
+  * @failed@ - gives higher priority to the tests which failed in the previous (local) build
+
+p(info). %In addition you can either decide to run the whole test suite (@ordering@ mode) or only those tests which fall into the selected categories (@selecting@ mode).%
+
+p(important). %@new@, @changed@ and @affected@ rely on SCM information. For this release we only support **Git**.%
+
+h3. How to get started
+
+p. If you are using Maven 3.3.x or newer (and you definitely should!) adding Smart Testing to your build is easy. Create a file `.mvn/extensions.xml` in the root of project 
+with the following content:
+
+div(filename). extensions.xml
+
+bc(prettify).. <?xml version="1.0" encoding="UTF-8"?>
+<extensions>
+  <extension>
+    <groupId>org.arquillian.smart.testing</groupId>
+    <artifactId>maven-lifecycle-extension</artifactId>
+    <version>0.0.1</version>
+  </extension>
+</extensions>
+
+p. and you are all set!
+
+p. Executing the build with Smart Testing enabled is as simple as adding one system property with strategies of your choice:
+
+@$ mvn clean verify -Dsmart.testing=new,changed,affected@
+
+This will execute the build with only those tests which are falling into those categories, based on local changes.

```

Comment:
Have you mentioned somewhere that the default mode is `selecting`? I think that it should be written either here or in the section describing modes. Or both. Otherwise, it is quite confusing IMHO...